.github/workflows/build-&-publish-docker-image.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       run: |
         if [[ "$PUSH_LATEST" == "yes" ]] && [[ "$BRANCH" == "master" ]]; then
```

Comment:
this could be simplified to 
docker push $MINER_REGISTRY:latest
no if statement needed right?

Reply:
no this should not be changed

Reply:
but there is no PUSH_LATEST variable anymore as that input is gone 
therefore this will need to be removed if we ever want to push master latest